paymentmethod.go

```diff
@@ -0,0 +1,185 @@
+package stripe
+
+import "encoding/json"
+
+// PaymentMethodType is the list of allowed values for the payment method type.
+type PaymentMethodType string
+
+// List of values that PaymentMethodType can take.
+const (
+	PaymentMethodTypeCard        PaymentMethodType = "card"
+	PaymentMethodTypeCardPresent PaymentMethodType = "card_present"
+)
+
+// PaymentMethodCardBrand is the list of allowed values for the brand property on a
+// Card PaymentMethod.
+type PaymentMethodCardBrand string
+
+// List of values that PaymentMethodCardBrand can take.
+const (
+	PaymentMethodCardBrandAmex       PaymentMethodCardBrand = "amex"
+	PaymentMethodCardBrandDiners     PaymentMethodCardBrand = "diners"
+	PaymentMethodCardBrandDiscover   PaymentMethodCardBrand = "discover"
+	PaymentMethodCardBrandJCB        PaymentMethodCardBrand = "jcb"
+	PaymentMethodCardBrandMastercard PaymentMethodCardBrand = "mastercard"
+	PaymentMethodCardBrandUnionpay   PaymentMethodCardBrand = "unionpay"
+	PaymentMethodCardBrandVisa       PaymentMethodCardBrand = "visa"
+	PaymentMethodCardBrandUnknown    PaymentMethodCardBrand = "unknown"
+)
+
+// PaymentMethodCardWalletType is the list of allowed values for the type a wallet can take on
+// a Card PaymentMethod.
+type PaymentMethodCardWalletType string
+
+// List of values that PaymentMethodCardWalletType can take.
+const (
+	PaymentMethodCardWalletTypeAmexExpressCheckout PaymentMethodCardWalletType = "amex_express_checkout"
+	PaymentMethodCardWalletTypeApplePay            PaymentMethodCardWalletType = "apple_pay"
+	PaymentMethodCardWalletTypeGooglePay           PaymentMethodCardWalletType = "google_pay"
+	PaymentMethodCardWalletTypeMasterpass          PaymentMethodCardWalletType = "masterpass"
+	PaymentMethodCardWalletTypeSamsungPay          PaymentMethodCardWalletType = "samsung_pay"
+	PaymentMethodCardWalletTypeVisaCheckout        PaymentMethodCardWalletType = "visa_checkout"
+)
+
+// PaymentMethodTransferDataParams is the set of parameters allowed for the transfer_data hash.
+type PaymentMethodTransferDataParams struct {
+	Destination *string `form:"destination"`
+}
```

Comment:
I think this is unintentional—this struct doesn't appear to be used anywhere.

Reply:
ah yes bad copy paste, fixed (won't push while you review)